.env

```diff
@@ -10,7 +10,7 @@ INSTALL_PLUGINS="sociallogin groupfolders"
+CONFIG_JSON='{ "system": { "auto_logout": false, "remember_login_cookie_lifetime": "0", "session_lifetime": "60*60*2", "session_keepalive": false, "upgrade.disable-web": true, "trashbin_retention_obligation": "7,7", "filelocking.enabled": false, "social_login_auto_redirect": true, "templatesdirectory": "", "skeletondirectory": "", "profile.enabled": false, "auth.webauthn.enabled": false, "allow_user_to_change_display_name": false, "upgrade.disable-web": false, "config_is_read_only": false, "app_install_overwrite": [ "gluusso", "schulcloud", "extract" ] }, "apps": { "core": { "shareapi_default_permission_cancreate": "no", "shareapi_default_permission_canupdate": "no", "shareapi_default_permission_candelete": "no", "shareapi_default_permission_canshare": "no", "shareapi_default_permissions": "1", "shareapi_restrict_user_enumeration_full_match": "no", "shareapi_allow_share_dialog_user_enumeration": "no", "shareapi_allow_group_sharing": "no", "shareapi_allow_resharing": "no", "shareapi_allow_public_upload": "no", "shareapi_allow_links": "no", "shareapi_enabled": "no" }, "files": { "types": "filesystem", "enabled": "yes", "default_quota": "100 GB" }, "schulcloud": { "enabled": "yes", "types": "" }, "extract": { "enabled": "yes", "types": "" }, "sociallogin": { "allow_login_connect": "", "auto_create_groups": "1", "create_disabled_users": "", "custom_providers": "{\"custom_oidc\":[{\"name\":\"SchulcloudNextcloud\",\"title\":\"SchulcloudNextcloud\",\"authorizeUrl\":\"http:\\/\\/localhost:9000\\/oauth2\\/auth\",\"tokenUrl\":\"http:\\/\\/hydra-host:4444\\/oauth2\\/token\",\"displayNameClaim\":\"name\",\"userInfoUrl\":\"http:\\/\\/hydra-host:4444\\/userinfo\",\"logoutUrl\":\"http:\\/\\/localhost:3100\\/dashboard\",\"clientId\":\"Nextcloud\",\"clientSecret\":\"Nextcloud\",\"scope\":\"openid offline profile email groups\",\"groupsClaim\":\"groups\",\"style\":\"none\",\"defaultGroup\":\"schulcloud\"}]}", "disable_notify_admins": "1", "disable_registration": "", "enabled": "yes", "hide_default_login": "true", "no_prune_user_groups": "", "prevent_create_email_exists": "", "restrict_users_wo_assigned_groups": "", "restrict_users_wo_mapped_groups": "", "types": "", "update_profile_on_login": "1" } } }'
```

Comment:
"auto_logout": false, "remember_login_cookie_lifetime": "0", "session_lifetime": "60*60*2", "session_keepalive": false, "upgrade.disable-web": true

This parameters are new.